src/Resources/config/config.xml

```diff
@@ -200,4 +200,16 @@
             </helpText>
         </component>
     </card>
+    <card>
+        <title>Domain URL</title>
```

Comment:
Can we add a enabled/disabled toggle for this config? For storefront sales channels we are able to get the active domain from the context, only headless sales channels need this option set

Reply:
Thanks for the review @peterojo, this is now added